internal/pkg/cmd/prerunner.go

```diff
@@ -29,6 +29,16 @@ import (
 	"github.com/confluentinc/cli/internal/pkg/version"
 )
 
+type wrongLoginCommandError struct {
+	errorString      string
+	suggestionString string
+}
+
+var wrongLoginCommandErrors = map[string]wrongLoginCommandError{
```

Comment:
I was thinking this mapping could just be from `confluent cluster` to `confluent kafka cluster`, not the whole suggestion. Odds are we'll want a similar suggestion for each mapping, and we don't want to have to write out the whole suggestion every time.

Reply:
that sounds better